test/test-golangci-lint.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# print current running test
+echo running "$0"
+
+command -v golangci-lint >/dev/null 2>&1 || { echo >&2 "golangci-lint not found"; exit 1; }
+
+# configure settings for test scripts
+ROOT=$(dirname "${BASH_SOURCE}")/..
+cd "${ROOT}" # Enter mgmt root
+. test/util.sh
+
+failures=''
+function run-test()
+{
+	$@ || failures=$( [ -n "$failures" ] && echo "$failures\\n$@" || echo "$@" )
+}
+
+# using .golangci.yml config file settings in ROOT
+gcl='golangci-lint run'
+
+# commented out from gometalinter linter test
```

Comment:
are these five lines of comments valid?

Reply:
Added these lines in case somebody were to move over any changes from gometalinter to golangci-lint. There were some linters which were entirely commented out, whereas some were still left as TODO, since they still require some fixes. More for clarity purposes, but they can be removed.

Reply:
Are they still valid?